src/wallet/wallet.cpp

```diff
@@ -4581,3 +4581,20 @@ ScriptPubKeyMan* CWallet::AddWalletDescriptor(WalletDescriptor& desc, const Flat
 
     return ret;
 }
+
+void CWallet::LoadWalletID(const uint160& id)
+{
+    wallet_id = id;
```

Comment:
Assert it's currently null? (set only once)

Reply:
Done